Tools/YamlCreate.ps1

```diff
@@ -107,7 +107,7 @@ Function Read-WinGet-MandatoryInfo {
 Function Read-PreviousWinGet-Manifest {
     Switch ($Option) {
         'Update' {
-            $LastVersion = Split-Path (Split-Path (Get-ChildItem -Path "$AppFolder\..\" -Recurse -Depth 1 -File).FullName ) -Leaf | Sort-Object $ToNatural | Select-Object -Last 1
+            $LastVersion = Split-Path (Split-Path (Get-ChildItem -Path "$AppFolder\..\" -Recurse -Depth 1 -File -Exclude ".validation").FullName ) -Leaf | Sort-Object $ToNatural | Select-Object -Last 1
```

Comment:
```suggestion
            $LastVersion = Split-Path (Split-Path (Get-ChildItem -Path "$AppFolder\..\" -Recurse -Depth 1 -File -Filter "*.yaml").FullName ) -Leaf | Sort-Object $ToNatural | Select-Object -Last 1
```

Oop, I made a mistake too; I forgot that `-Exclude` overrides the `-Depth` as we found in [Trenly#11](https://github.com/Trenly/winget-pkgs/pull/11#issuecomment-896984783) . Based on [this commit](https://github.com/Trenly/winget-pkgs/pull/11/commits/696ab0e7a92bd20426c2c65dce35265256acc477), I think this should work now